src/main/java/kuit/subway/controller/StationController.java

```diff
@@ -0,0 +1,42 @@
+package kuit.subway.controller;
+
+import kuit.subway.request.PostStationRequest;
+import kuit.subway.response.GetStationsResponse;
+import kuit.subway.response.PostStationResponse;
+import kuit.subway.service.StationService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+@RequestMapping("/stations")
+public class StationController {
+    private final StationService stationService;
+
+    @PostMapping
+    public ResponseEntity<PostStationResponse> createStation(
+            @Validated @RequestBody PostStationRequest postStationRequest){
+        PostStationResponse postStationResponse = stationService.createStation(postStationRequest.getName());
+        return new ResponseEntity<>(postStationResponse, HttpStatus.CREATED);
```

Comment:
다음과 같은 방식이 좀 더 깔끔해보이는데 어떤가요?? 

```suggestion
       return ResponseEntity.created(URI.create("/stations/" + station.getId())).body(station);
```

Reply:
ResponseEntity를 custom하는 BaseResponseEntity를 만들었는데,
Step2 때 함께 pr보내도록 하겠습니다 ㅎㅎㅎ